.curlrc

```diff
@@ -0,0 +1,7 @@
+--continue-at -
+--include
+--location
+--retry 5
+--retry-delay 60
+--retry-max-time 1800
+--silent
```

Comment:
Shouldn't this (and `$CURL_HOME`) be provided by our build images instead?

Reply:
good question, yes it could be a better place for it, I'll check if I can solve this

Reply:
On one side, putting the `.curlrc` file in the build image, in the `$HOME` directory, or setting `$CURL_HOME` globally, removes the need to patch many shell scripts to take it into account or to explicitly pass `--config "$(git rev-parse --show-toplevel)"` to all the `curl` commands that are invoked outside of those shell scripts.

On the other side, having the `.curlrc` file in the build image means it will be slightly more painful to update and it will guarantee that same commands won’t have the same behavior in the CI and on the developers’ local environment.

I can see pros and cons for both approaches. I’m not sure which one is the best.